app/presenters/return-requirements/check/returns-requirements.presenter.js

```diff
@@ -56,17 +64,30 @@ function _mapPurposes (requirementPurposes, purposes) {
   })
 }
 
-function _mapRequirement (requirement, index, purposes) {
+function _mapRequirement (requirement, index, purposes, points) {
   return {
     abstractionPeriod: _abstractionPeriod(requirement.abstractionPeriod),
     frequencyCollected: requirement.frequencyCollected,
     frequencyReported: requirement.frequencyReported,
     index,
+    points: _mapPoints(requirement.points, points),
     purposes: _mapPurposes(requirement.purposes, purposes),
     siteDescription: requirement.siteDescription
   }
 }
 
+function _mapPoints (requirementPoints, points) {
+  if (!requirementPoints || !points) {
+    return []
+  }
```

Comment:
I don't think we need to check for this.

If the licence has no points the user wouldn't have made it past the `/points` page, which also means the return requirement is incomplete.

Assuming the licence has points, again the user wouldn't have got past this page without having selected at least one.

So, we only call `_mapRequirement()` for completed return requirements, which means it is okay for the code to assume both `requirementPoints` and `points` are populated.

```suggestion
```